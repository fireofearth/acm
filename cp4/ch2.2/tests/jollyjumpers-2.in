4 1 4 2 3
5 1 4 2 -1 6
1 100
3 1 2 0
3 1 2 1
3 1 2 -1
