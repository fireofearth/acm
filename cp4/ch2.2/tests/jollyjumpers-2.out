Jolly
Not jolly
Jolly
Jolly
Not jolly
Not jolly
